.github/workflows/checks - race.yml

```diff
@@ -0,0 +1,32 @@
+name: Quick Checks (race)
+
+on:
+  workflow_dispatch: 
+
+jobs:
+  race-tests:
```

Comment:
Do we need this dedicated workflow?  Can this be removed now that the intermittent issue has been identified?

Reply:
I figured it would be useful for the review process, but you're right, I can remove it. I'll just past a copy in the related issue so that if we ever face a similar hard-to-reproduce issue, we can simply add it back to the repo, run it, then remove it

Reply:
Workflow was deleted as per feedback.